ports/stm32/boards/stm32h723_af.csv

```diff
@@ -1,116 +1,138 @@
+PortB,PB15,RTC_REFIN                  ,TIM1_CH3N                   ,TIM12_CH2                 ,TIM8_CH3N                                     ,USART1_RX                                                   ,I2S2_SDO/SPI2_MOSI                  ,DFSDM1_CKIN2                               ,                                           ,UART4_CTS                                       ,SDMMC2_D1                                               ,                                              ,                                                                        ,FMC_D11/FMC_DA11                     ,                               ,LTDC_G7         ,EVENTOUT,                                                                 
+PortC,PC0 ,                           ,FMC_D12/FMC_DA12            ,                          ,DFSDM1_CKIN0                                  ,                                                            ,                                    ,DFSDM1_DATIN4                              ,                                           ,SAI4_FS_B                                       ,FMC_A25                                                 ,USB_OTG_HS_ULPI_STP                           ,LTDC_G2                                                                 ,FMC_SDNWE                            ,                               ,LTDC_R5         ,EVENTOUT,ADC1_INP10/ADC2_INP10/ADC3_INP10                                 
+PortC,PC1 ,DEBUG_TRACED0              ,SAI4_D1                     ,SAI1_D1                   ,DFSDM1_DATIN0                                 ,DFSDM1_CKIN4                                                ,I2S2_SDO/SPI2_MOSI                  ,SAI1_SD_A                                  ,                                           ,SAI4_SD_A                                       ,SDMMC2_CK                                               ,OCTOSPIM_P1_IO4                               ,ETH_MDC                                                                 ,MDIOS_MDC                            ,                               ,LTDC_G5         ,EVENTOUT,ADC1_INN10/ADC1_INP11/ADC2_INN10/ADC2_INP11/ADC3_INN10/ADC3_INP11
+PortC,PC2 ,                           ,                            ,                          ,DFSDM1_CKIN1                                  ,OCTOSPIM_P1_IO5                                             ,I2S2_SDI/SPI2_MISO                  ,DFSDM1_CKOUT                               ,                                           ,                                                ,OCTOSPIM_P1_IO2                                         ,OTG_HS_ULPI_DIR/USB_OTG_HS_ULPI_DIR           ,ETH_MII_TXD2                                                            ,FMC_SDNE0                            ,                               ,                ,EVENTOUT,ADC1_INN11/ADC1_INP12/ADC2_INN11/ADC2_INP12/ADC3_INN11/ADC3_INP12
```

Comment:
This is missing the dual-pad ADCs for H7 (added in the boardgen PR).

Reply:
dual-pad chip support has been added now.